src/variables/variable_common.jl

```diff
@@ -52,9 +52,33 @@ function add_variable!(
             ind => _variable(m, name, ind, lb, ub, bin, int)
             for ind in indices(m; t=vcat(history_time_slice(m), time_slice(m)))
         )
+    if use_representative_days(model=m.ext[:instance]) == :use_representative_days_w_seasonal_storage
+        map_variable!(m, m.ext[:variables][name], indices)
+    end
     ((bin != nothing) || (int != nothing)) && push!(m.ext[:integer_variables], name)
 end
 
+"""
+    map_variable!(v::Dict{VariableRef}, indices_all::Function)
+
+Extends variable Dictionary such that indexing by a non representative
+time slice returns the variable with the corresponding representative
+time slice.
+"""
+function map_variable!(m::Model, var::Dict, var_indices::Function)
+    # TODO: should probably rename "ordering_periods" to
+    # "time_slice_mapping"
```

Comment:
I think `representative_periods` is a nice name. "Ordering" doesn't sound correct to me, sounds like something else.

`representative_periods` or `representative_time_slices` sounds good. `time_slice_mapping` is too general and doesn't say anything about 'representative'.